dfs-diffusers/docker/gpu/Dockerfile

```
@@ -11,7 +11,7 @@ RUN pip3 install --no-cache-dir -r requirements.txt
 
 COPY download.py .
 
-RUN python3 download.py --model $MODEL_ID --refiner $REFINER_ID
+RUN python3 download.py --model $MODEL_ID
```

I think this is the only change required, the rest can be reverted?

```suggestion
RUN python3 download.py --model $MODEL_ID --refiner "$REFINER_ID"
```

---

@casperdcl then for [this](https://github.com/premAI-io/prem-services/blob/3d132f84093f3503d1012a62ef3cf97a423c06e9/dfs-diffusers/build.sh#L10) image build REFINER_ID won't be used correctly by download.py to download refiner only when given. Whereas it'll work for all the other image builds.

---

oh wait I totally missed that you've put REFINER_ID inside quotes, my bad! damn this totally works then

---

just pushed a commit (might make it clearer what I mean) - feel free to revert it if it doens't make sense or doesn't work though. Basically:

- missing `--build-arg="REFINER_ID=..."`
  + `Dockerfile` executes `download.py --refiner ""`
    * `argparse` defaults to `args.refiner = None`
- missing `--build-arg="REFINER_ID=..."` -> s
  + `Dockerfile` executes `download.py --refiner "..."`
    * `argparse` parses into `args.refiner = "..."`

---

thanks for the explanation! this makes sense, just tried locally and seems to be working nicely.